src/components/Card/Card.js

```diff
@@ -3,7 +3,7 @@ import './Card.css';
 function Card(props) {
   return (
     <li className='card'>
-      <img className='card__image' src={props.card.image} alt='#'/>
+      <div className={`card__image card__image_type_${props.type}`} style={{backgroundImage: `url(${props.card.image})`}}></div>
```

Comment:
should we specify type of card for classname, since other things like the carousel items i'm making could be understood as "card"? feels like it should be a bit more unique. maybe `column-card` for example?

Reply:
I think that's a good idea, but it would be better to call the cards for the carousel `carousel-card` and to leave these as `card` in my opinion.

Reply:
It seems slightly different than what I typically think of as "cards" in ux eng, but overall makes sense if we want to consider it our primary card! Feel free to resolve this comment :)